pythran/pythonic/numpy/fromfile.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef PYTHONIC_NUMPY_FROMSTRING_HPP
+#define PYTHONIC_NUMPY_FROMSTRING_HPP
+
+#include "pythonic/include/numpy/fromfile.hpp"
+
+#include "pythonic/types/list.hpp"
+#include "pythonic/types/ndarray.hpp"
+#include "pythonic/types/str.hpp"
+#include "pythonic/utils/functor.hpp"
+#include <stdio.h>
+
+#include <limits>
+#include <sstream>
+
+PYTHONIC_NS_BEGIN
+
+namespace numpy
+{
+  template <class dtype>
+  types::ndarray<typename dtype::type, types::pshape<long>>
+  fromfile(types::str const &file_name, dtype d, long count,
+           types::str const &sep, long offset)
+  {
+    // open file as read binary and find its length
```

Comment:
if you only support binary, you should raise a NotImplemented Error. when sep is not empty